src/yaxarrays/YAXArrays.jl

```diff
@@ -0,0 +1,36 @@
+import Base.copy
+
+function axes_names(cube::YAXArray)
+    return name.(cube.axes)
+end
+
+"""
+    sort(cube::YAXArray, axis_name::Symbol)
+
+Sorts axis labels of a given YAXArray datacube.
+"""
+function sort(cube::YAXArray, axis_name::Symbol)::YAXArray
+    axis_labels = collect(lookup(cube, axis_name))
+    labels_sort_idx::Vector{Int64} = sortperm(axis_labels)
+    ordered_labels = axis_labels[labels_sort_idx]
+
+    # Get selector to order data
+    _axes_names = axes_names(cube)
+    n_axes = length(_axes_names)
+    selector::Vector{Union{Colon,Vector{Int64}}} = fill(:, n_axes)
+    axis_idx::Int64 = findfirst(x -> x == axis_name, _axes_names)
+    selector[axis_idx] = labels_sort_idx
+
+    return cube[selector...]
+end
+
+"""
+    copy(cube::YAXArray)::YAXArray
+
+Use julia's Base.copy function to create a copy of an YAXArray
+"""
+function copy(cube::YAXArray)::YAXArray
+    new_data = copy(cube.data)
+    new_axlist = Tuple(ax for ax in deepcopy(cube.axes))
+    return YAXArray(new_axlist, new_data)
+end
```

Comment:
Hmm, you're assigning a copy of the data to a variable then passing it into YAXArray.

If my understanding of Julia is correct, it'll temporarily allocate double the memory, and then at some point it will trigger the GC.

I think this is better (and more concise, without losing readability):

```julia
function copy(cube::YAXArray)::YAXArray
    new_axlist = Tuple(ax for ax in deepcopy(cube.axes))
    return YAXArray(new_axlist, copy(cube.data))
end
```